lib/workers/repository/update/pr/changelog/release-notes.ts

```diff
@@ -171,7 +171,7 @@ async function releaseNotesResult(
   }
   // set body for release notes
   releaseNotes.body = massageBody(releaseNotes.body, baseUrl);
-  if (releaseNotes.body.length) {
+  if (releaseNotes.body.length || releaseNotes.name?.length) {
```

Comment:
We need to filter when release name equals version, which would duplicate the version and would happen for sure

Reply:
Yeah, strip out version prefix, trim it, and then include it only if text remains